Add blog post about London meetup (#63)
openwebdocs/owd-website: pages/content/posts/london-meetup/index.md

---
author: "Florian Scholz"

> title: "Open Web Docs meetup with Mozilla’s MDN Team in London"

Reviewer: Consider dropping "in London", just for brevity.

Author: ```suggestion
title: "Open Web Docs meetup with Mozilla’s MDN Team"
```

> date: "2023-02-08"

Author: ```suggestion
date: "2023-02-09"
```

tags: ["announcements"]
ShowToc: false
ShowBreadCrumbs: false
excerpt: "On January 18-20 all of the Open Web Docs Technical Writing team and members of the OWD Governing Committee were invited to London to meet with Mozilla’s MDN Team."
---

On January 18-20 all of the Open Web Docs Technical Writing team and members of the OWD Governing Committee were invited to London to meet with Mozilla’s MDN Team. Mozilla kindly sponsored our travel and accommodation and Google hosted all of us in their London office. We are very grateful for this opportunity. It was the first time we met as a group in person given Open Web Docs was founded during the global covid pandemic. Some of us met for the first time in real life.

For Open Web Docs, the most important project we contribute to is MDN Web Docs. We work closely with Mozilla, our main sponsors Google, Microsoft, Igalia, JetBrains, Canva, our 150+ individual funders, and the larger web community to maintain and innovate on the best documentation site for web developers and designers world wide.

> At the end of last year, we started to work more closely with Mozilla than ever before to help with community coordination and to keep the MDN contribution process running efficiently. To that end, Mozilla and OWD also created a [shared and open roadmap](https://github.com/orgs/mdn/projects/26/) for ongoing and planned work. The Open Web Docs team is the largest group of people that contributes to MDN projects (mainly content and BCD) with the highest number of contributions and we are happy to collaborate and give direct input on MDN’s information architecture, compatibility data, content structures, code examples, and other changes coming up across the whole site.

Reviewer: "Largest group of people" or "largest team" - is it the intention to highlight the size of the team rather than the size of the contributions? It doesn't seem terribly relevant (and depending on how we count, not necessarily true).

> At the end of last year, we started to work more closely with Mozilla than ever before to help with community coordination and to keep the MDN contribution process running efficiently. To that end, Mozilla and OWD also created a [shared and open roadmap](https://github.com/orgs/mdn/projects/26/) for ongoing and planned work. The Open Web Docs team is the largest group of people that contributes to MDN projects (mainly content and BCD) with the highest number of contributions and we are happy to collaborate and give direct input on MDN’s information architecture, compatibility data, content structures, code examples, and other changes coming up across the whole site.

Author: I'm omitting the sentence. Within the context of this paragraph it is pretty clear we are a big deal and no need to be comparative. 

```suggestion
In Q4 of 2022, Open Web Docs started working very closely with Mozilla on community coordination and keeping the MDN contribution process running efficiently and effectively. To that end, Mozilla and OWD created a [shared and open roadmap](https://github.com/orgs/mdn/projects/26/) for ongoing and planned work. The Open Web Docs team actively collaborates and gives direct input on MDN’s information architecture, compatibility data, content structures, code examples, and other changes coming impacting all sections of the MDN Web Docs site.
```

> With well over 10,000 pages on MDN and an ever evolving Web platform to document, the scope, requirements, and the impact of changes for web developers all around the world can be quite a challenge to get right. With all of the expertise from the different organizations, we all come together on MDN Web Docs to provide web developers with information they need to create amazing web pages and applications. It is this mission that drives all of us to collaborate closely, share knowledge, and bring new ideas to the table.

Reviewer 2: ```suggestion
With well over 11,000 pages on MDN Web Docs and an ever-evolving Web platform to document, the scope, requirements, and impact of changes for web developers all around the world can be quite a challenge to get right. With all of the expertise from the technical writer and advisory committee teams, we are all coming together on MDN Web Docs to provide web developers with the information they need to create accessible, secure, and performant web pages and applications. It is this mission that drives all of us to collaborate closely, share knowledge, and bring new ideas to the table.
```

> Our shared roadmap introduces a new tool that will help us to capture significant site changes, from content updates to navigation improvements, and will empower Mozilla, OWD, and our community to implement and execute complex projects together.

Reviewer: ```suggestion
Our shared roadmap will help us to capture significant site changes, from content updates to navigation improvements, and will empower Mozilla, OWD, and our community to implement and execute complex projects together.
```

> Many of the discussions in London were about how to collaborate more closely. We also had a few technical deep dive sessions where we were able to discuss in person some of the current projects we are working on. The minutes for these sessions are [available on our GitHub](https://github.com/openwebdocs/project/blob/main/steering-committee/meetups/london-2023/index.md). The topics were:

Reviewer 2: ```suggestion
Many of the discussions in London were about how to collaborate more closely. We also had a few technical deep dive sessions where we were able to discuss in person some of the current projects we are working on. The minutes for these sessions are [available on our GitHub](https://github.com/openwebdocs/project/blob/main/steering-committee/meetups/london-2023/index.md). The areas of focus are:
```

* [Content lifecycle](https://github.com/openwebdocs/project/blob/main/steering-committee/meetups/london-2023/session-content.md): We identified the maintenance cost of content additions during planning, defined sunsetting strategies, and discussed ways to separate technical or implementation details from prose that may quickly become outdated.

> * [Information architecture / page types](https://github.com/openwebdocs/project/blob/main/steering-committee/meetups/london-2023/session-ia.md): We talked through the ongoing work to categorize and organize content on the site based on the type of technology it relates to, and how it works with other technologies.

Reviewer 2: ```suggestion
* [Information architecture / page types](https://github.com/openwebdocs/project/blob/main/steering-committee/meetups/london-2023/session-ia.md): We talked through the ongoing work to categorize and organize content on the site based on the technology it relates to, and how it works with other technologies.
```

---
author: "Florian Scholz"
title: "Open Web Docs meetup with Mozilla’s MDN team"

> date: "2023-02-09"

Author: ```suggestion
date: "2023-02-08"
```
